conf/tariffLinks.conf

```diff
@@ -26,6 +26,8 @@ urls {
   tariff.declaration.ducr.clearance.0 = "https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-c21-customs-clearance-request-completion-guide-inventory-exports/group-2-references-of-messages-document-certificates-and-authorisations#de-21-simplified-declarationprevious-documents-box-40-declarationprevious-document"
   tariff.declaration.lrn.common.0 = "https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-export-declaration-completion-guide/group-2-references-of-messages-document-certificates-and-authorisations#de-25-lrn-no-previous-reference"
   tariff.declaration.lrn.clearance.0 = "https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-c21-customs-clearance-request-completion-guide-inventory-exports/group-2-references-of-messages-document-certificates-and-authorisations#de-25-lrn-no-previous-reference"
+  tariff.declaration.linkDucrToMucr.common.0 = "https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-export-declaration-completion-guide/group-2-references-of-messages-document-certificates-and-authorisations#de-21-simplified-declaration-previous-documents-box-40-declaration-previous-document"
```

Comment:
Isn't this the same as the link in L31?

Reply:
Yes but we need a different key for the Link-to-Mucr page